### PR TITLE
Synthengine

### PR DESCRIPTION
**Pull request by PhilippAubert**
Add the synth-engine for the app. 
AmpEnvelope is a blocker

### CODE REVIEW

File: digitalsynth/src/components/css/Footer.css [hunk 1]
```diff
@@ -1,10 +1,11 @@
 .FooterNav {
-  width: 100vw;
+  width: 89vw;
```
Comment (marwinburesch) [inline digitalsynth/src/components/css/Footer.css:2]: This should be handled directly by its parent element's flexbox. You have some classes competing in App.css and here, one should be enough

File: digitalsynth/src/components/css/Footer.css [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
   display: flex;
   flex-direction: row;
   justify-content: space-around;
   align-items: center;
   margin: 0 auto;
-  font-family: "Press Start 2P";
-  font-size: 1.3vh;
+  font-family: "Toxigenesis";
+  font-size: 1.8vh;
```
Comment (marwinburesch) [inline digitalsynth/src/components/css/Footer.css:9]: `vh` is not a good unit for font-size
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

**Review by marwinburesch — CHANGES_REQUESTED** [review]
This is impressive mate, it feels awesome how interactive this app already is. Good work!
The logic for all this sound magic looks a bit wild, but I think it is excellent, and it works 💯  

There are a few points I'd like to talk about:
- Your `App.js` feels difficult to handle, I think it's time to create some components and move code to other files, especially those `<svg>`s. [Check this out for some inspiration](https://github.com/marwinburesch/capstone/blob/661727c34a50a56384501623c124310a0e0de029/src/App.js#L5)
- Your next step should be moving basic markup from App.js to components. All those sliders and
- The way reactrouter is implemented is only working partially as you intended
  - You really only need one `<Router>` component in your `App.js` or `index.js` in order to have components rendered in your app depending on the `url`.
  - This is already working in your `App.js`, however your `<Footer>` doesn't need it. The `<Footer>` doesn't do anything yet, and in the future it will probably only need to provide links to other pages, not routes
- And finally, this PR is simply too big. It promises "Synthengine" and delivers a lot more 😛, please try to focus PRs on less issues 

You've made so much progress in just a week with this project, just add a little project management and it'll feel even smoother :)
cheers

File: digitalsynth/src/App.js [hunk 3]
```diff
@@ -197,42 +330,110 @@ export default function App() {
...
+            </button>
+            <button className="OnOff" onClick={onClickStop}>
+              {" "}
+              Stop{" "}
+            </button>
+          </div>
+        </main>
+        <footer className="App-Footer">
+          <Footer />
+        </footer>
```
[the remark this answers is not in the document]
Comment (PhilippAubert) [reply digitalsynth/src/App.js:432]: All moved into the new components 😄

**Pull request closed without merging**